README.md~4226a885f776162d97188ccd8407ef58d15299ec

```diff
@@ -0,0 +1,3 @@
+# todo-app
```

Comment:
@shak-m It seems you created a second README. Why?

Reply:
I'm not sure how this happened, I've deleted one of the README files.